### ISSUE TITLE
Support <script>-cloaked templates

### ISSUE THREAD

**Issue by fortes** [Treesaver/treesaver#258]
Related to #257

**Comment by bramstein** [comment]
I'll take it :)

**Comment by bramstein** [comment]
This has been implemented per SHA: ee683faa5c243dc77fbc7847a03cb933360721fe

Cloaked `<script>` elements can be used as follows:

```
<script type="text/html" data-ts-template="toc">
  ...
</script>
```

The `<script>` element will be replaced by a `div` element once the template has been expanded.
